handler/handler_suite_test.go

```diff
@@ -19,10 +19,9 @@ import (
 
```

Comment:
just a nit .. Add the sufix _test to the package. ( handler_test )

Reply:
reason:  the suffix `_test` which has [special meaning for the Go compiler](https://github.com/golang/go/issues/25223).